info_terminal/infremen/src/infremen.cpp

```diff
@@ -97,9 +98,10 @@ uint32_t getMidnightTime(uint32_t givenTime)
 //parameter reconfiguration
 void reconfigureCallback(infremen::infremenConfig &config, uint32_t level) 
 {
-	ROS_INFO("Reconfigure Request: %lf %d %d %d %d", config.explorationRatio, config.minimalBatteryLevel, config.interactionTimeout, config.maxTaskNumber, config.taskDuration);
+    ROS_INFO("Reconfigure Request: %lf %d %d %d %d %d", config.explorationRatio, config.minimalBatteryLevel, config.batteryUndockLevel, config.interactionTimeout, config.maxTaskNumber, config.taskDuration);
```

Comment:
Can I ask you to make sure you use the same indentation as in the rest of the file as a courtesy to the original author?